internal/nix/search.go

```diff
@@ -94,6 +94,14 @@ func pkgExistsForAnySystem(pkg string) bool {
 }
 
 func searchSystem(url string, system string) map[string]*Info {
+	// Search will download nixpkgs if it's not already downloaded. Adding this
+	// check here provides a slightly better UX.
+	if IsGithubNixpkgsURL(url) {
+		hash := HashFromNixPkgsURL(url)
+		// TODO, pass legit writer, handle error
```

Comment:
could we fix this prior to landing, otherwise we'll likely forget

Reply:
I added some comments on why stderr is OK and why we want to ignore the error. Passing in a writer is a bit complicated. I think longer term we can restructure the nix package to have an struct. Another option is to hard code everything to stderr.